### DISCUSSION TITLE
getting-latex-working-in-vscode-on-windows/

### DISCUSSION THREAD

**Discussion by giscus** [JakeSteam/blog-programming#43] [Comments]
# getting-latex-working-in-vscode-on-windows/

I recently decided to learn the basics of LaTeX, a way of preparing documents primarily used in STEM academia. I also decided to use VSCode, as I do for almost all other work. This was not as easy as I expected, so here’s a complete beginner’s guide to getting LaTeX working in Visual Studio Code on Windows!

https://blog.jakelee.co.uk/getting-latex-working-in-vscode-on-windows/

**Comment by jvfd3** [comment]
Thank you for the tutorial

**Comment by schreibmachine** [comment]
This tutorial was a lifesaver. My goodness gracious, WHY is it so complicated?! I concur with all your thoughts in the last section. Alas, the only way to improve open-source is... ah well, my brain is too small to install LaTeX properly, so I shan't. Thank you for the very accurate and detailed instructions! May your tea be always be perfect drinking temperature.

**Comment by shenpengfii** [comment]
Brilliant man! Sad to see if any AI product gets feed on it. This tutorial actually deserves a reward. Also, an other section for using `TinyTex` with vscode on windows os (as a greatly less storage consuming plan) will be much more better for our budget.
